completions/bash/multipass

```diff
@@ -40,10 +40,80 @@ _multipass_complete()
         done
     }
 
+    # Set $opts to the list of available networks.
+    _multipass_networks()
+    {
+        local cmd="multipass networks --format=csv 2>/dev/null"
+
+        opts=$( \eval $cmd | \grep -Ev '(\+--|Name)' | \cut -d',' -f 1 )
```

Comment:
Little suggestion for improvement: if this comes out empty, do not show any auto completion opts (since nothing can be accomplished with `--network`). Right now, if the daemon is down or if have a driver that does not support bridging, I still get `id= mac= mode=`.

Reply:
Good, will also implement it.

Reply:
Done.